doc/source/whatsnew/v0.21.1.txt

```diff
@@ -54,40 +83,57 @@ Documentation Changes
 
 .. _whatsnew_0211.bug_fixes:
 
-Bug Fixes
```

Comment:
I think we need a section header here  for all the bug fixes (can do in followup)

Reply:
That should go in a normal PR to master, we should not do any fixes here (apart from possible merge conflicts)

Reply:
Ah, that was my fault. I messed up a merge conflict I think when cherry picking.

Reply:
ah, yes, this one indeed seems like a wrong merge